### DISCUSSION TITLE
Web Docs

### DISCUSSION THREAD

**Discussion by afifurrohman-id** [oliver-oloughlin/kvdex#41] [Ideas]
First of all thanks for creating this library and very helpful for me / other people as newbie in Deno. because is simple to understand like  using orm

**May be any plans to create the web docs?**
Some people need another example like how to's create a simple todo app and more,

thank you..

**Comment by oliver-oloughlin** [comment]
Hi, thanks for showing interest in kvdex, I really appreciate it!

I hadn't actually thought about creating any web docs, as I've tried to keep the README quite extensive and up to date. But if this could be helpful to you and others like you say I would be glad to do it at some point in the near future. I don't have a lot of free time, but I can definitely look into it.

What sort of examples were you thinking of that would be most useful? There is of course the approach of full web documentation, like this: https://nextjs.org/docs. Or it could be in the form of a blog post/tutorial going through all the steps in creating some example application, like this: https://faun.pub/creating-a-basic-webserver-in-deno-using-oak-eee3e8d9b998. Which do you prefer?

**Reply by roberto-morado** [reply]
I'll be more than happy to help

**Reply by oliver-oloughlin** [reply]
Seeing as I have never created any documentation before, that would be very much appreciated!
